crates/uv-cache/src/lib.rs

```diff
@@ -793,7 +793,7 @@ impl CacheBucket {
         match self {
             // Note that when bumping this, you'll also need to bump it
             // in crates/uv/tests/cache_prune.rs.
-            Self::SourceDistributions => "sdists-v6",
+            Self::SourceDistributions => "sdists-v7",
```

Comment:
This is a bit of a bummer, since we're invalidating a TON of data. If we don't do this, though, then users running newer versions can hit these errors if they then check out older versions (since we'd be writing to `sdists-v6`, but those old versions would fail to read, since these errors are fatal right now).

It's a tradeoff. I'm undecided on what's best (bump or leave it as-is).

Reply:
I don't have strong feelings here. This feels vaguely correct. Are there other cache changes we've been delaying?

Reply:
I don't believe so...

Reply:
I think it's right to bump here. Correctness and errors should be given priority.